scala/compiler-jps/src/org/jetbrains/jps/incremental/scala/data/CompilerData.scala

```diff
@@ -155,5 +155,11 @@ object CompilerData extends CompilerDataFactory {
     versions.exists(version.startsWith)
   }
 
+  def compilerVersion(module: JpsModule): Option[String] = compilerJarsIn(module) match {
+    case Some(CompilerJars(_, compiler, _)) => version(compiler)
+    case None => None
+  }
+  def version(compiler: File): Option[String] = readProperty(compiler, "compiler.properties", "version.number")
```

Comment:
`readProperty(compiler, "compiler.properties", "version.number")` is being used in the `versionIn` method. Now you can call `version` instead.

The rest looks fine to me.

Reply:
I have squashed the change, and given that it's published API I also renamed `version(compiler: File)` to `compilerVersion`. Did I understand your request correctly?

Reply:
Yes. Cool, thanks.